dockstore-webservice/src/main/java/io/dockstore/webservice/core/Collection.java

```diff
@@ -56,7 +57,7 @@
     @ApiModelProperty(value = "Description of the collection", position = 2)
     private String description;
 
-    @JsonIgnore
+    @JsonIgnoreProperties("collections")
```

Comment:
I think this is overbroad.
In terms of best practice, this should be a class level annotation, that way it would be less unexpected if a new collection was added.
e.g. https://www.concretepage.com/jackson-api/jackson-jsonignore-jsonignoreproperties-and-jsonignoretype#JsonIgnoreProperties

However, in this case @JsonIgnore seems more appropriate if the intent was just to ignore "entries".

Reply:
The intent was to include the entries but to not include the collections field for the entries.

Reply:
Oh, in that case, it might be better to experiment with lazy loading on the collections field in the Entry class.

Reply:
Lazy loading doesn't seem to work.